__tests__/app.test.js

```diff
@@ -70,3 +71,30 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+
+describe("GET /api/articles", () => {
+  test("gives status of 200 and responds with all articles", () => {
```

Comment:
As we're checking the order in which the articles are sorted in this test, we should also include this in the description 👍